src/password/Forgot.vue

```diff
@@ -72,6 +74,7 @@ export default {
           this.$router.push(`/password/forgot/${reset.uid}`)
         } catch (error) {
           grecaptcha.reset()
+          eventBus.emit('error', error)
```

Comment:
even with the new error handling in main.js it still wasn't picking up this and I didn't find an easy way to pick up all error from methods if we want to. see: https://vuejs.org/api/application#app-config-errorhandler

Reply:
Don't we want to show errors on all pages? Even if the user may not understand them, at least the message can be passed on to a support agent or to us directly.

Reply:
I've implemented the most global way to catch error I can find in Vue 3, and I think it will continue to catch most things, but something about this method, maybe it being async or something else is not letting it propagate without directly sending it to the bus. I will have to research more to find out why.

Reply:
I pushed a change so we don't have to emit an event when there is a caught error, only re-throw it in the catch

Reply:
Good solution